src/Analysis/Ast/Impl/Types/PythonClassType.cs

```diff
@@ -38,12 +38,26 @@ internal class PythonClassType : PythonType, IPythonClassType, IPythonTemplateTy
         private Dictionary<string, IPythonType> _genericParameters;
         private string _documentation;
 
+        /// <summary>
+        /// Creates function for specializations
+        /// </summary>
+        public static PythonClassType Specialize(string name, IPythonModule declaringModule, string documentation, bool isAbstract = false) {
```

Comment:
Does class specialization actually needs `isAbstract`? I think we determine it from methods/bases, right?

Reply:
This is to specialize abc.ABC as an abstract class (sort of the root abstract class)